setup.py

```diff
@@ -26,6 +26,7 @@
     'recommonmark>=0.4.0',
     'Sphinx>=1.3.5',
     'sphinxcontrib-napoleon>=0.4.4',
+    'sphinx-rtd-theme>=0.1.9',
```

Comment:
I found an old issue related to that https://github.com/rtfd/readthedocs.org/issues/173, I'm asking to reopen it.